src/Organizations/OrganizationForm/OrganizationSummaryForm/OrganizationSummaryForm.js

```diff
@@ -77,8 +77,8 @@ function OrganizationSummaryForm({ initialValues }) {
     onResetVendorFields();
   }, [onResetVendorFields]);
 
-  const onChangeIsVendor = useCallback(({ target: { checked } }) => {
-    change('isVendor', checked);
+  const handleCheckbox = useCallback(({ target: { checked, name } }) => {
+    change(name, checked);
 
     if (initialValues.id && !checked) setVendorUncheckConfirm(true);
```

Comment:
With this generic function that handles both is-vendor and is-donor (er, actually, it can handle any checkbox), this will prompt a confirmation modal for an org that is a donor but not a vendor unchecks the is-donor box. Is this the desired behavior?